### PR TITLE
[HCO] cover release-1.11

### PR DESCRIPTION
**Pull request by dharmit**

### CODE REVIEW

**Comment by dharmit** [comment]
- Used go1.20 in image names
- Used OCP 4.14 wherever 4.13 was used in old PR
- Replaced 1.10 occurrences with 1.11 and 1.9 occurrences with 1.10

PTAL if I have misconfigured anything.

/cc @nunnatsa 
/cc @tiraboschi

**Comment by nunnatsa** [comment]
/pj-rehearse pull-ci-kubevirt-hyperconverged-cluster-operator-release-1.11-hco-e2e-upgrade-prev-operator-sdk-aws

File: ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-release-1.11.yaml [hunk 1]
```diff
@@ -0,0 +1,756 @@
...
+- as: hco-e2e-upgrade-prev-operator-sdk-sno-azure
+  optional: true
+  run_if_changed: ^(api/.*|assets/.*|build/.*|ci-test-files/.*|config/.*|controllers/.*|cmd/.*|deploy/crds/.*|deploy/index-image/.*|hack/.*|pkg/.*|tests/.*|Makefile|go\.mod|go\.sum)$
+  steps:
+    cluster_profile: azure-virtualization
+    dependencies:
+      OO_BUNDLE: hco-bundle-reg-prev
+    env:
+      BASE_DOMAIN: cnv-devel.azure.devcluster.openshift.com
+      INITIAL_VERSION: 1.11.0
```
Comment (nunnatsa) [inline ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-release-1.11.yaml:254]: Should be `1.10.0`

File: ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-release-1.11.yaml [hunk 2]
```diff
@@ -0,0 +1,756 @@
...
+- as: hco-e2e-upgrade-prev-operator-sdk-sno-aws
+  optional: true
+  run_if_changed: ^(api/.*|assets/.*|build/.*|ci-test-files/.*|config/.*|controllers/.*|cmd/.*|deploy/crds/.*|deploy/index-image/.*|hack/.*|pkg/.*|tests/.*|Makefile|go\.mod|go\.sum)$
+  steps:
+    cluster_profile: aws-virtualization
+    dependencies:
+      OO_BUNDLE: hco-bundle-reg-prev
+    env:
+      BASE_DOMAIN: cnv-ci.syseng.devcluster.openshift.com
+      INITIAL_VERSION: 1.11.0
```
Comment (nunnatsa) [inline ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-release-1.11.yaml:393]: Should be `1.10.0`

File: ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-release-1.11.yaml [hunk 3]
```diff
@@ -0,0 +1,756 @@
...
+    workflow: optional-operators-ci-operator-sdk-azure
+- as: hco-e2e-upgrade-prev-operator-sdk-aws
+  run_if_changed: ^(api/.*|assets/.*|build/.*|ci-test-files/.*|config/.*|controllers/.*|cmd/.*|deploy/crds/.*|deploy/index-image/.*|hack/.*|pkg/.*|tests/.*|Makefile|go\.mod|go\.sum)$
+  steps:
+    cluster_profile: aws-virtualization
+    dependencies:
+      OO_BUNDLE: hco-bundle-reg-prev
+    env:
+      BASE_DOMAIN: cnv-ci.syseng.devcluster.openshift.com
+      INITIAL_VERSION: 1.11.0
```
Comment (nunnatsa) [inline ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-release-1.11.yaml:654]: Should be `1.10.0`

File: ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-release-1.11.yaml [hunk 4]
```diff
@@ -0,0 +1,756 @@
...
+    workflow: optional-operators-ci-operator-sdk-aws
+- as: hco-e2e-upgrade-prev-operator-sdk-azure
+  run_if_changed: ^(api/.*|assets/.*|build/.*|ci-test-files/.*|config/.*|controllers/.*|cmd/.*|deploy/crds/.*|deploy/index-image/.*|hack/.*|pkg/.*|tests/.*|Makefile|go\.mod|go\.sum)$
+  steps:
+    cluster_profile: azure-virtualization
+    dependencies:
+      OO_BUNDLE: hco-bundle-reg-prev
+    env:
+      BASE_DOMAIN: cnv-devel.azure.devcluster.openshift.com
+      INITIAL_VERSION: 1.11.0
```
Comment (nunnatsa) [inline ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-release-1.11.yaml:708]: Should be `1.10.0`

**Review by nunnatsa — CHANGES_REQUESTED** [review]
Please fix "upgrade-prev" lanes. Added inline comments to point where to fix

**Comment by nunnatsa** [comment]
/pj-rehearse pull-ci-kubevirt-hyperconverged-cluster-operator-main-okd-hco-e2e-upgrade-operator-sdk-aws

File: ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-main__okd.yaml [hunk 5]
```diff
@@ -1,10 +1,10 @@
 base_images:
   base:
-    name: "4.14"
+    name: "4.15"
```
Comment (tiraboschi) [inline ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-main__okd.yaml:3]: Please do this in a separate PR since it's focusing on main

Comment (dharmit) [reply ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-main__okd.yaml]: Removed it from the okd.yaml file.
